Sources/Extensions/UIColor.swift

```diff
@@ -19,10 +19,12 @@ extension UIColor {
 
         var rgb: UInt32 = 0
 
+        // swiftlint:disable identifier_name
         var r: CGFloat = 0.0
```

Comment:
Personally, I'm not a huge fan of abbreviation. Instead to disable the error. We should just rename r as red, green as green, b as blue and a as alpha.